### DISCUSSION TITLE
essay/2024/03/19/resume-github-blog

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#197] [Comments]
# essay/2024/03/19/resume-github-blog

10년 전에 이력서를 받을 땐 GitHub과 블로그 주소를 적어둔 이력서를 좋아했습니다. GitHub에서 직접 코드를 볼 수 있고 얼마나 코딩에 진심인지도 알 수 있으니까. 블로그도 좋은 레퍼런스였습니다. 어떤 기술에 관심이 있고 어느 정도 깊이로 이해하고 있는지를 봤습니다.

https://jeho.page/essay/2024/03/19/resume-github-blog.html

**Comment by sieu-n** [comment]
Goodhart's law is an adage often stated as, "When a measure becomes a target, it ceases to be a good measure"
